bentoml/yatai/client/bento_repository_api.py

```diff
@@ -195,12 +205,29 @@ def upload_from_dir(self, saved_bento_path, labels=None):
             )
 
         if response.uri.type == BentoUri.LOCAL:
-            if os.path.exists(response.uri.uri):
-                # due to copytree dst must not already exist
-                shutil.rmtree(response.uri.uri)
-            shutil.copytree(saved_bento_path, response.uri.uri)
+            # For remote yatai service, call upload method.
+            if is_remote_yatai(self.yatai_service):
+                self._upload_bento(
+                    bento_service_metadata.name,
+                    bento_service_metadata.version,
+                    saved_bento_path,
+                )
+                update_bento_service = UpdateBentoRequest(
+                    bento_name=bento_service_metadata.name,
+                    bento_version=bento_service_metadata.version,
+                    service_metadata=bento_service_metadata,
+                )
+                self.yatai_service.UpdateBento(update_bento_service)
```

Comment:
why do we need it here?

Reply:
We combined update upload status and bento metadata in the `_update_bento_upload_progress` call.  For remote yatai with FS repo, we will need to make sure the metadata is updated.